FunctionalTableDataTests/TableSectionsValidationTests.swift

```diff
@@ -37,7 +37,12 @@ class TableSectionsValidationTests: XCTestCase {
 		}, failure: {
 			XCTAssertEqual($0.name, NSExceptionName.internalInconsistencyException)
 			XCTAssertNotNil($0.userInfo as? [String: Any])
-			XCTAssertEqual($0.userInfo!["Duplicates"] as? Set<String>, Set(arrayLiteral: "section1"))
+			
+			if let duplicates = $0.userInfo?["Duplicates"] as? [String] {
+				XCTAssertEqual(Set(duplicates), Set(["section1"]))
```

Comment:
isn't this always going to make them the same? if duplicates is `["section1", "section1"]` it will be collapsed to `["section1"`]

Reply:
`duplicates` is expected to contain each unique key that was duplicated: one entry per key, not one per section, but we don't care about the number. What we really care about is "does duplicates contain the duplicate key, with no false positives for other keys?"